rust/main/hyperlane-core/src/traits/pending_operation.rs

```diff
@@ -156,7 +156,6 @@ pub trait PendingOperation: Send + Sync + Debug + TryBatchAs<HyperlaneMessage> {
     fn reset_attempts(&mut self);
 
     /// Set the number of times this operation has been retried.
-    #[cfg(any(test, feature = "test-utils"))]
```

Comment:
this was sorta needed bc of the new resolver, otherwise some things wouldn't compile. Didnt want to deal with the headache of it so figured we can just add it